swagger-templates/src/main/java/com/okta/swagger/codegen/AbstractOktaJavaClientCodegen.java

```diff
@@ -338,13 +340,39 @@ private void handleOktaLinkedOperations(Swagger swagger) {
                 swagger.getPaths().forEach((pathName, path) -> {
                     Optional<Map.Entry<HttpMethod, Operation>> operationEntry =
                         path.getOperationMap().entrySet().stream().filter(
-                            e -> e.getValue().getOperationId() != null &&
-                                e.getValue().getOperationId().equals(operationId)).findFirst();
+                            oper -> {
+                                //Looking for operationId in paths:path:operationId
+                                if (oper.getValue().getOperationId() != null
+                                    && oper.getValue().getOperationId().equals(operationId)) {
+                                    return true;
+                                }
+                                //Looking for operationId in paths:path:method:x-okta-multi-operation:operationId
+                                Object multiOperation = oper.getValue().getVendorExtensions().get("x-okta-multi-operation");
+                                if (multiOperation instanceof List) {
+                                    Optional<JsonNode> jsonNodeOptional = ((List)multiOperation).stream().filter(x -> {
+                                        if(x instanceof JsonNode) {
+                                            JsonNode operationIdFromMultiOperation = ((JsonNode)x).get("operationId");
```

Comment:
Here is an unchecked assignment for a generic type. Also, you should not do `findFirst()`, and then `isPresent()`: it could be replaced with `anyMatch()`.
Could be simplified to this:
```
    if (multiOperation instanceof List) {
        return ((List<?>) multiOperation).stream()
            .filter(element -> element instanceof JsonNode)
            .map(jsonNode -> ((JsonNode) jsonNode).get("operationId"))
            .filter(Objects::nonNull)
            .anyMatch(operationIdFromMultiOperation -> operationIdFromMultiOperation.textValue().equals(operationId));
    }
```